pod/perlguts.pod

```diff
@@ -3021,6 +3021,16 @@ You can use this to concatenate two scalars:
  SV *var3 = newSVpvf("var1=%" SVf " and var2=%" SVf,
                      SVfARG(var1), SVfARG(var2));
 
+=for apidoc Amnh||SVf_QUOTEDPREFIX
+
+C<SVf_QUOTEDPREFIX> is similar to C<SVf> except that it restricts the
+number of the characters printed, showing at most the first
+C<PERL_QUOTEDPREFIX_LEN> characters of the argument, and rendering it with
+double quotes and with the contents escaped using double quoted string
+escaping rules. If the string is longer than this then elipses "..."
+will be appended after the trailing quote. This is intended for error
+messages where the string is assumed to be a class name.
```

Comment:
"or method"... or presumably some other bits and pieces as well..? Might be worth trying to be a bit more general in the wording here.

Reply:
done

Reply:
yep, done.